packages/reporter/src/controllers/SygmaReporterController.ts

```diff
@@ -0,0 +1,38 @@
+import { Chain, parseEther } from "viem"
```

Comment:
is `SygmaReporterController` used?

Reply:
Deleted `SygmaReporterController.js` and use `StandardReporterController` for Sygma reporter: 5e9b58a1f0d7618f1b5ab57d6b43a2fa144b0d59